src/components/20-molecules/top-content-bar/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 5.0.0
+
+- BREAKING CHANGE: If you provided `<axa-link>` as child to the component, you
```

Comment:
In my opinion this shouldn't be a task of the user

Reply:
We shouldn't have given the user the option to do both. Not sure if SEO was the reason for that, because button-link is created by the top content bar.

But ok, I will rewrite this component to use a child-css for this margin.